README.md

```diff
@@ -1,16 +1,14 @@
-Google App Engine Managed VMs Python Compat Runtime
+Google App Engine Flexible Environment Python Compat Runtime
 ===================================================
 
-Warning
--------
-The instructions here are for a multi-process version of the Python 2.7 runtime
-**currently in alpha**. To use the stable version of the runtime, follow the
```

Comment:
Wait - so the previous python27 runtime was beta, and we changed the default in gcloud to drop a dockerfile for an image in alpha?